### PR TITLE
Feature/sql helper

### PR DESCRIPTION
**Pull request by EfiShimon**

### CODE REVIEW

File: awesome_provider/app/src/provider_model.py [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
 
-class provider_moderl:
-    def __init__(self , url, usr, password, dbName):
-        self.query = QueryHelper(url, usr, password, dbName)
+class provider_model:
+    #def __init__(self , url, usr, password, dbName):
+     #   self.query = QueryHelper(url, usr, password, dbName)
 
-    def CheckHealth(self):
+    def CheckHealth():
```
Comment (asihamza) [inline awesome_provider/app/src/provider_model.py:7]: IF this is a method that belongs to the class, you have to pass `self`

File: awesome_provider/app/src/provider_routing.py [hunk 2]
```diff
@@ -3,23 +3,24 @@
...
+#    dbPass = sys.argv[2]
+#    dbUser = sys.argv[3]
+#    dbName = sys.argv[4]
+#    dbPort = sys.argv[5]
 
-    model = provider_model(dbUrl, dbUser, dbPass, dbName, dbPort)
-    app.run(host='0.0.0.0',debug=True)
\ No newline at end of file
+    #model = provider_model(dbUrl, dbUser, dbPass, dbName, dbPort)
+app.run(host='0.0.0.0',debug=True)
```
Comment (asihamza) [inline awesome_provider/app/src/provider_routing.py]: `debug`?

File: awesome_provider/docker-compose.yml [hunk 3]
```diff
@@ -13,15 +14,23 @@ services:
...
+            
+            
     sql:
+        container_name: mysql
         build: ./db_scripts
         ports: 
-            - "3306:3306"
-        volumes:
-            - '${PWD}/conrainer_data:/docker-entrypoint-initdb.d'
+            - "3306:3306"
```
Comment (asihamza) [inline awesome_provider/docker-compose.yml:23]: Volume?!

**Review by asihamza — CHANGES_REQUESTED** [review]
You forgot volume in docker-compose.yml && some python comments.

File: awesome_provider/app/src/provider_model.py (hunk 1 above)
Comment (EfiShimon) [reply awesome_provider/app/src/provider_model.py:7]: using the method as static method will fix in  later version

File: awesome_provider/app/src/provider_routing.py (hunk 2 above)
Comment (EfiShimon) [reply awesome_provider/app/src/provider_routing.py]: will remove

File: awesome_provider/docker-compose.yml (hunk 3 above)
Comment (EfiShimon) [reply awesome_provider/docker-compose.yml:23]: added on next commit

**Review by asihamza — APPROVED** [review]

**Pull request merged**